src/Agent/NewRelic/Agent/Core/Api/AgentApi.cs

```diff
@@ -300,7 +317,8 @@ void work()
         /// Supports web applications only. 
```

Comment:
I notice you didn't add this... This sounds like a bug though vs something to document, why would it only work for web applications?

Reply:
NoticeError should work in any type of application.